tensorflow/python/training/saver.py

```diff
@@ -1390,20 +1390,20 @@ def import_meta_graph(meta_graph_or_file,
   NOTE: Restarting training from saved `meta_graph` only works if the
   device assignments have not changed.
 
-  Example 2:
+  Example:
   Variables, placeholders, and independent operations can also be stored, as
   shown in the following example.
 
   ```Python
   # Saving contents and operations.
   v1 = tf.compat.v1.placeholder(tf.float32, name="v1")
   v2 = tf.compat.v1.placeholder(tf.float32, name="v2")
-  v3 = tf.mul(v1, v2)
+  v3 = tf.multiply(v1, v2)
```

Comment:
This should be `tf.math.multiply`. Thank you for catching the change!

Reply:
Thank you Paige. One question. Do we need to add `math` for all math ops? I noticed math ops in many examples using tf.multiply, tf.add etc. I think it is best to use `math` and I will follow as you suggested for any future PRs (including this PR). Thanks again.

Reply:
@dynamicwebpaige Please let me know if anything missing. Thanks!